### ISSUE TITLE
AVT 1 - Form Item: DAP violations related to label

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#2459]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed Description
Run DAP with Feb. 2019 ruleset on the Form Item Component
The following violations are reported:

Each form control must have an associated label
H44: Using label elements to associate text labels with form controls
ARIA16: Using aria-labelledby to provide a name for user interface controls
Provide descriptive text in label element
4.1.2 Name, Role, Value

This issue reopens #2188 

To fix the issue fix the typo by replacing arialabel="Numeric input field with increment and decrement buttons" with aria-label="Numeric input field with increment and decrement buttons".
Remove the <label> or include some text in the label. (See screenshot).
<img width="1211" alt="Screen Shot 2019-05-08 at 8 20 27 AM" src="https://user-images.githubusercontent.com/21676914/57378641-d4130880-716a-11e9-90e0-c9f0ca3a6795.png">

**Comment by snidersd** [comment]
@dakahn I just retested and confirmed that there are no DAP violations.
